BlueprintUICommonControls/Tests/Sources/XCTestCaseExtensions.swift

```diff
@@ -30,7 +30,8 @@ extension XCTestCase {
 
 
         let testName = testName.prefix(testName.count - 2)
-        imageURL.appendPathComponent("\(testName)_\(identifier ?? "").png")
+        let majorVersion = UIDevice.current.systemVersion.components(separatedBy: ".")[0]
```

Comment:
`ProcessInfo.processInfo.operatingSystemVersion` gives you a `majorVersion` property to avoid the string parsing here.

Reply:
Nice, will update!